pom.xml

```diff
@@ -178,7 +180,7 @@
     <version.org.eclipse.emf.ecore.xmi>2.5.0.v20100521-1846</version.org.eclipse.emf.ecore.xmi>
     <version.org.eclipse.jdt>3.18.0</version.org.eclipse.jdt>
     <!-- Jetty's version is aligned with CXF -->
-    <version.org.eclipse.jetty>9.4.53.v20231009</version.org.eclipse.jetty>
+    <version.org.eclipse.jetty>9.4.44.v20210927</version.org.eclipse.jetty>
```

Comment:
This is a downgrade, but 2.11 is using the older version `9.4.44.v20210927` . 
IINW, `jetty` doesn't land in the runtime artifacts, so we should be safe.